test/e2e-go/features/incentives/mining_test.go

```diff
@@ -163,17 +188,31 @@ func TestBasicPayouts(t *testing.T) {
 		block, err := client.BookkeepingBlock(status.LastRound)
 		a.NoError(err)
 
-		feesink := block.BlockHeader.FeeSink
-		err = fixture.WaitForRoundWithTimeout(status.LastRound + 1)
-		a.NoError(err)
-		data, err := client.AccountData(feesink.String())
+		a.EqualValues(bonus1, block.Bonus.Raw)
+
+		data, err := client.AccountData(block.Proposer().String())
 		a.NoError(err)
-		fmt.Printf(" feesink has %d at round %d\n", data.MicroAlgos.Raw, status.LastRound)
+		fmt.Printf(" proposer %v has %d after proposing round %d\n", block.Proposer(), data.MicroAlgos.Raw, status.LastRound)
+
+		feesink := block.BlockHeader.FeeSink
+		// show all the node's belief about feesink, for debugging the payout
+		// effects appearing locally but not elsewhere (or vice versa)
+		for i, c := range []libgoal.Client{relay, c01, c15} {
+			data, err = c.AccountData(block.Proposer().String())
+			a.NoError(err)
+			a.Equal(block.Round(), data.LastProposed)
+
+			data, err = c.AccountData(feesink.String())
+			a.NoError(err)
+			fmt.Printf(" feesink %d has %d at round %d\n", i, data.MicroAlgos.Raw, status.LastRound)
+		}
 		a.LessOrEqual(100000, int(data.MicroAlgos.Raw)) // won't go below minfee
```

Comment:
Should this go inside the above for loop? Otherwise you're only asserting on c15's view of the feesink.

Reply:
I suppose.  The latter two for-loops are kind of silly.  They print things, but do no comparisons.  They are probably overkill, and should be deleted.  If I keep it, I'll assert the feesink has the same value in each, then it won't matter where this line goes.